lib/matplotlib/projections/polar.py

```diff
@@ -115,15 +115,30 @@ def inverted(self):
 
 
 class PolarAffine(mtransforms.Affine2DBase):
-    """
-    The affine part of the polar projection.  Scales the output so
-    that maximum radius rests on the edge of the axes circle.
+    r"""
+    The affine part of the polar projection.
+
+    Scales the output so that maximum radius rests on the edge of the axes
+    circle and the origin is mapped to (0.5, 0.5). The transform applied is
+    the same to x and y components and given by:
+
+    .. math::
+
+        x_{1} = 0.5 \left [ \frac{x_{0}}{(r_{max} - r_{min})} + 1 \right ]
+
+    :math:`r_{min}, r_{max}` are the minimum and maximum radial limits after
```

Comment:
```suggestion
        x_{1} = 0.5 \left [ \frac{x_{0}}{(r_{\max} - r_{\min})} + 1 \right ]

    :math:`r_{\min}, r_{\max}` are the minimum and maximum radial limits after
```
to get slightly better formatting.